recipes/aws-noaa-solar-irradiance-daily/recipe.py

```diff
@@ -0,0 +1,35 @@
+from os.path import join
+import s3fs
+from pangeo_forge_recipes.patterns import pattern_from_file_sequence
+from pangeo_forge_recipes.recipes.reference_hdf_zarr import HDFReferenceRecipe
+
+s3_bucket = 'noaa-cdr-total-solar-irradiance-pds'
+s3_files_path = 'data/daily/'
+url_base = f's3://{s3_bucket}/{s3_files_path}'
+file_list = []
+fs = s3fs.S3FileSystem(anon=True)
+
+
+def is_nc(x):
+    return x.endswith('.nc')
+
+
+def add_s3(x):
+    return 's3://' + x
+
+
+file_list = sorted([add_s3(nc_file) for nc_file in fs.ls(join(url_base)) if is_nc(nc_file)])
+pattern = pattern_from_file_sequence(file_list, 'time', nitems_per_file=1)
+recipe = HDFReferenceRecipe(pattern, netcdf_storage_options={'anon': True})
```

Comment:
this recipe results in a ds with only a time dim but total irradiance varies by location along the top of atmosphere https://en.wikipedia.org/wiki/Solar_irradiance

we need to extract the coordinate information which appears to be denoted by this text in each file

tsi_v02r01-preliminary_daily_s20220101_e20220331_c20220419.nc
tsi_v02r01

Reply:
@rbavery , I could not find anyway to get the coordinates and values, only i found a file that have `; time (yyyy-MM-dd), TSI (W m-2), uncertainty (W m-2)` values  `s3://noaa-cdr-total-solar-irradiance-pds/data/ancillary-data/tsi-ssi_v02r01_observed-tsi-composite_s19780101_e20220930_c20221018.txt`  also from the files names is not possible to get them , do you have any other suggestion to get the coordinates?

Reply:
I think from the data documentation, there is no spatial coordinate: 

https://www.ncei.noaa.gov/pub/data/sds/cdr/CDRs/Solar%20Spectral%20Irradiance/CDR_Flyer_01B-33.pdf